src/mlpack/core/data/split_data.hpp

```diff
@@ -18,11 +18,108 @@
 namespace mlpack {
 namespace data {
 
+template<typename InputType, typename LabelsType>
+void SplitHelper(const InputType& input,
+           const LabelsType& inputLabel,
+           InputType& trainData,
+           InputType& testData,
+           LabelsType& trainLabel,
+           LabelsType& testLabel,
+           const double testRatio,
+           const bool shuffleData = true)
```

Comment:
```suggestion
                const LabelsType& inputLabel,
                InputType& trainData,
                InputType& testData,
                LabelsType& trainLabel,
                LabelsType& testLabel,
                const double testRatio,
                const bool shuffleData = true)
```
Just a spacing fix. :)  Also, it would be great to add a comment here describing what the function does (even though it's an 'internal' function---the comments are still very helpful for maintenance).